src/Ozds.Client/App.razor

```diff
@@ -1,10 +1,6 @@
-<HeadOutlet @rendermode="RenderMode.InteractiveServer"/>
```

Comment:
ovo nam treba - nezz zas si maknuo headoutlet

Reply:
aha to je već od prije. Dodao